variables.tf

```diff
@@ -121,19 +123,20 @@ variable "bucket_location" {
 
 variable "auto_create_network" {
   description = "Create the default network"
-  default     = "false"
+  type = bool
```

Comment:
Not related to your change, please revert

Reply:
Reverted